ReactAndroid/src/main/java/com/facebook/react/modules/camera/CameraRollManager.java

```diff
@@ -248,6 +252,7 @@ public void getPhotos(final ReadableMap params, final Promise promise) {
     private final int mFirst;
     private final @Nullable String mAfter;
     private final @Nullable String mGroupName;
+    private final @Nullable String mAssetType;
```

Comment:
Remove `@Nullable` from here because your code below expects it's always "Photos" or "Videos".